client/jetpack-cloud/sections/settings/advanced-credentials/credentials-form/index.tsx

```diff
@@ -102,24 +105,82 @@ const ServerCredentialsForm: FunctionComponent< Props > = ( { onCredentialsSave,
 		</FormFieldset>
 	);
 
+	const getSubHeaderText = () => {
+		if ( hostInfo !== null && hostInfo.inline !== undefined ) {
+			return translate( 'Check the information icons for details on %(hostName)s', {
+				args: {
+					hostName: hostInfo?.name,
+				},
+			} );
+		} else if ( hostInfo !== null && hostInfo.supportLink !== undefined ) {
+			return translate(
+				'Read through the {{a}}%(hostName)s support site{{/a}} to learn how to obtain your credentials.',
+				{
+					args: {
+						hostName: hostInfo.name,
+					},
+					components: {
+						a: <a target="_blank" rel="noopener noreferrer" href={ hostInfo.supportLink } />,
+					},
+				}
+			);
+		}
+		return translate( 'Your hosting provider will be able to supply this information to you.' );
+	};
+
+	const renderCredentialLinks = () => {
+		if ( hostInfo !== null && hostInfo.credentialLinks !== undefined ) {
+			if ( 'ftp' === formState.protocol && hostInfo.credentialLinks.ftp !== undefined ) {
+				return (
+					<Button href={ hostInfo.credentialLinks.ftp } target="_blank">
+						{ translate( 'Read the %(hostName)s credentials guide', {
+							args: {
+								hostName: hostInfo.name,
+							},
+						} ) }
+						<Gridicon icon="external" />
+					</Button>
+				);
+			}
+			if ( 'ssh' === formState.protocol && hostInfo.credentialLinks.sftp !== undefined ) {
+				return (
+					<Button href={ hostInfo.credentialLinks.sftp } target="_blank">
+						{ translate( 'Read the %(hostName)s credentials guide', {
+							args: {
+								hostName: hostInfo.name,
+							},
+						} ) }
+						<Gridicon icon="external" />
+					</Button>
+				);
+			}
+		}
+		return null;
+	};
+
 	return (
 		<div className="credentials-form">
 			<h3>{ translate( 'Provide your SSH, SFTP or FTP server credentials' ) }</h3>
-			<FormFieldset>
+			<p>{ getSubHeaderText() }</p>
+			{ renderCredentialLinks() }
+			<FormFieldset className="credentials-form__protocol-type">
 				<div className="credentials-form__support-info">
-					<FormLabel htmlFor="protocol-type">{ translate( 'Credential Type' ) }</FormLabel>
-					{ hostInfo && hostInfo.credentialType && (
+					<FormLabel htmlFor="protocol-type">{ translate( 'Credential type' ) }</FormLabel>
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.